### DISCUSSION TITLE
SSL error? Could not connect to tracker - QNAP TS-119 arm

### DISCUSSION THREAD

**Discussion by falkenhawk** [transmission/transmission#8382] [Support]
Hi, I have problems running torrents with this version on my QNAP TS-119 (arm processor).
I tried a package from https://www.forum-nas.fr/viewtopic.php?f=21&t=2148 and https://www.qnapclub.eu/en/qpkg/208 (it's the same package), also tried with transmission from the new merged entware - https://github.com/Entware/Entware/wiki/Using-Transmission

One tracker works fine over https, but the other does not. I checked it on a transmission/win on my pc and it works there so it's probably not an issue of a tracker I am trying to connect to.

I set `export TR_CURL_VERBOSE=1` and `export TR_DEBUG_FD=2`

and run transmission with 
```
transmission-daemon  --config-dir /share/HDA_DATA/.qpkg/QTransmission/etc --logfile /share/HDA_DATA/.qpkg/QTransmission/etc/transmission.log --log-debug -f 2>/share/HDA_DATA/.qpkg/QTransmission/etc/runlog.log
```
to save detailed debug log.

For the tracker which fails I found the following entry in the runlog
```
* error:0407006A:rsa routines:RSA_padding_check_PKCS1_type_1:block type is not 01
* Closing connection 5
```

this probably points to SSL handshake error.
I would be grateful for any help. I have no idea what I could do to fix this error.

**Comment by zebrasonmygrass** [comment]
Bumping this, I have the same issue. Transmission 3.00 on Debian 18.04 cannot handle SSL trackers, and there are no apparent measures I can take to work around this issue by omitting certificate verification.

**Comment by shenhanc78** [comment]
Same here Transmission 3.00 cannot connect to https trackers. transmission-daemon was built against libcurl4-gnutls-dev package.
